processes-endpoint.md

```diff
@@ -140,32 +140,144 @@ In a future version, a solution supporting a live-feed can also be included.
 ## Execution File Output List
 **GET /api/system/processes/<:process-id>/files**
 
-This endpoint will let an administrator download an output file created by a process. If the file is found, it will be presented as a download. This endpoint will support "Range" HTTP headers so that downloads can be paused and resumed.
-
+This endpoint will return a list of files that are associated with the process, this can be files uploaded by the end user for imports or files generated by the script like for exports of data.
+The files are grouped by "type", the type can be anything but they refer to what the file represents, is a mapping file, an exported csv file, ...
 ```json
 {
   "processId" : "4",
-  "_links" : {
-    "self" : {
-      "href" : "/api/system/processes/4/files"
+  "_embedded": {
+    "exportCSV": {
```

Comment:
should this be an array? can we assume that all scripts will accept and return only a single file for type?

Reply:
This can indeed be an array.
At the moment I don't know of a script using multiple files of the same type (e.g. multiple export CSVs), but it may be a use case that can occur in the future, and it's probably a small change